packages/closer/components/custom-pages/CustomListing.tsx

```
@@ -0,0 +1,116 @@
+            >
+              {item.title}
+            </Heading>
+
+            {settings?.isAccommodations ? (
+              <div className="text-md leading-normal">
+                {getFirstParagraph(item.text)}
+              </div>
+            ) : (
+              <div
+                className="rich-text text-md leading-normal"
+                dangerouslySetInnerHTML={{ __html: item.text }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)